servicecontrol/servicecontrol-in-practice.md

```diff
@@ -0,0 +1,28 @@
+
+For example, in a development environment you will want a lot of logging information to support problem analysis. For many users the [Debug Session](/servicecontrol/plugins/debug-session.md) plugin is really useful at this stage of the application life-cycle but you would never put that plugin into production. For the same reason, Particular advise that customers not use the [Saga Audit](/servicecontrol/plugins/saga-audit.md) plugin outside of development.
+
+Making decisions about the use of the other plugins and features requires a little more thought to balance the smooth running of a system with actual business requirements. The temptation to 'just use them all' as a catch-all insurance policy is not a good choice. The cost of having some of these features in play can actually cause issues later on if you are not prepared.
+
+[Auditing](/nservicebus/operations/auditing.md) is a good example of how the cost of having a record of everything may cause significant impact on the system. It might be worth considering turning audit off in some or all endpoints in production. Or if you absolutely need it move the auditing to another place like a Business Intelligence database or somewhere that has better resources dedicated to the problem. Remember audits in ServiceControl are primarily for performing system analysis using ServiceInsight. If you are not using ServiceInsight in production then do not use audits in production without having good reason.
+
+[Heartbeats](/servicepulse/intro-endpoints-heartbeats.md) and [Custom Checks](/servicepulse/intro-endpoints-custom-checks.md) are great for knowing when an endpoint is up or down but they add extra noise. Often, not all endpoints are mission critical. Therefore it is worth making you sure you understand and examine the risks to the system if an endpoint is not available for a while. The default heartbeat is forty seconds; maybe ten second updates are right for some endpoints but two minute intervals could be better for others. Remember also that all communication from endpoints with ServiceControl is performed via messaging. Adding a message to the queue every second may have little impact on when the notification shows up in ServicePulse. Remember, the last heartbeat is really the only one you care about. If you stop ServiceControl those heartbeats are going to bank up in the queue. You might want to consider flushing the heartbeats from the queue before starting ServiceControl again. Having a lot of noisy messages that add no value may overwhelm ServiceControl and cause it fail.
+
+##### Less can be more
+
+Moving forward, take a thoughtful approach in the adoption of plugins and features:
+
+- [Turn off auditing](http://docs.particular.net/nservicebus/operations/auditing#turning-off-auditing) on all endpoints as well as [heartbeats and custom checks](http://docs.particular.net/servicepulse/how-to-configure-endpoints-for-monitoring).
```

Comment:
u cant suggest the turn off audit. how about u suggest sending it to a queue no monitored by SC?